tests/testing.nightly

```diff
@@ -0,0 +1,154 @@
+//! A test for every supported item.
+//! This test requires the nightly toolchain to compile.
```

Comment:
Note: This file is named the way it is so that `cargo test` ignores it (so you don't need nightly installed to compile)